.circleci/config.yml

```diff
@@ -435,8 +431,11 @@ workflows:
 
   automatic-release:
     when:
-      not:
-        equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+      and:
+        - not:
+            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+        - not:
+            equal: [ dependency-update, << pipeline.parameters.action >> ]
```

Comment:
Hmm I wonder if this might be error prone in case we add other actions in the future...